src/actions/actions.js

```diff
@@ -0,0 +1,36 @@
+/* @flow */

```

Comment:
Move this to the module.

Also, I think that you should reuse class made by @OleksiyRudenko for API calls